### PR TITLE
create a button on the login page

### PR DESCRIPTION
**Pull request by v8j**

### CODE REVIEW

File: Registration Module Files/resources/scripts/components/auth/LoginContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    )}
+                    <div css={tw`mt-6 text-center`}>
+                        <Link
+                            to={'/auth/password'}
+                            css={tw`text-xs text-neutral-500 tracking-wide no-underline uppercase hover:text-neutral-600`}
+                        >
+                            Forgot password?
+                        </Link>
+                        <Link 
+                            to={'/auth/register'}
```
Comment (PadowYT2) [inline Registration Module Files/resources/scripts/components/auth/LoginContainer.tsx:110]: Needs to refresh the page the page after clicking

**Review by PadowYT2 — CHANGES_REQUESTED** [review]

File: Registration Module Files/resources/scripts/components/auth/LoginContainer.tsx (hunk 1 above)
Comment (PadowYT2) [reply Registration Module Files/resources/scripts/components/auth/LoginContainer.tsx:110]: Try adding
```js
const reloadpage = document.getElementById('reloadpage')
reloadpage.onclick = () => {
  window.location.reload();
}
```
and make put an id of `reloadpage` in the `Link`

**Comment by PadowYT2** [comment]
No response, closing

**Comment by v8j** [comment]
<__<

**Pull request closed without merging**